.github/workflows/community-pr.yml

```diff
@@ -28,16 +28,17 @@ jobs:
                 authorAssociation
               }
             }
-          }' -f org=$ORGANIZATION -f repo=$REPOSITORY -f pull="$PULL_REQUEST_NUMBER" -f label="$LABEL_NAME" > data.json
+          }' -f org=$ORGANIZATION -f repo=$REPOSITORY -F pull=$PULL_REQUEST_NUMBER -f label="$LABEL_NAME" > data.json
 
+        cat data.json
```

Comment:
📝 If the **Assign label** step is ever skipped, the log will not show the reason why it was skipped (which means you also cannot see the value of `AUTHOR_ASSOCIATION`). Logging this data will allow quick corrections if ever needed.